public/components/common/header.tsx

```diff
@@ -4,7 +4,7 @@
  */
 
 import React from 'react';
-import { EuiText, EuiLink, EuiPanel, EuiTitle } from '@elastic/eui';
+import { EuiText, EuiLink, EuiPanel, EuiTitle, EuiSpacer } from '@elastic/eui';
```

Comment:
Is this necessary?

Reply:
ah, that was a left over, reverting. Good catch.